packages/extension-ui/src/components/SplashHandler.tsx

```diff
@@ -86,7 +100,24 @@ function SplashHandler({ children, className }: SplashHandlerProps): React.React
   );
 }
 
-export default styled(SplashHandler)`
+const WrappedSplashHandler = ({ children, className }: Omit<SplashHandlerProps, 'isSplashShown'>) => {
+  const isSplashThrottled = useIsSplashThrottled();
+
+  if (isSplashThrottled === undefined) {
+    return null;
```

Comment:
Do I understand correctly that in theory there might be a blip of the splash not showed until the value is fetched from the storage, although that would practically never be visible, as the promise will pretty much always resolve before the component even renders?

Reply:
yes, I hope that loading a value from local storage (file) will be faster than loading a pretty big file (animation) and the blip won't ever be noticeable